src/graph_serializer.cc

```diff
@@ -31,8 +38,7 @@ namespace detail {
 			// Iterate over first level of dependencies.
 			// These might either be data transfer commands, or task commands from other tasks.
```

Comment:
This comment is no longer true, right?

Reply:
Right! It wasn't before either, I guess. Adjusted.